chia/util/initial-config.yaml

```diff
@@ -77,6 +77,8 @@ network_overrides: &network_overrides
       # planned 2.0 release is July 26, height 2965036 on testnet
       # 1 week later
       HARD_FORK_HEIGHT: 2997292
+      # November 2023
```

Comment:
2023?

Reply:
yes, these are the overrides for `testnet10`. The hard fork and chip-13 have already activated on testnet